### PR TITLE
GLTF actor colors from material

### PR DESCRIPTION
**Pull request by xtanion**
This small PR adds the ability to add colors to the actor from `materials` in the glTF file.

Example: Run the `BrainStem` model in `viz_gltf.py`
viz_gltf

Other models with material colors:
- BoxAnimated
- RiggedSimple
- [2 Cylinder Engine](https://github.com/KhronosGroup/glTF-Sample-Models/blob/master/2.0/2CylinderEngine)
- [Water Bottle](https://github.com/KhronosGroup/glTF-Sample-Models/blob/master/2.0/WaterBottle)
- [Buggy](https://github.com/KhronosGroup/glTF-Sample-Models/blob/master/2.0/Buggy) and many more.

### CODE REVIEW

File: fury/gltf.py [hunk 1]
```diff
@@ -169,6 +171,7 @@ def load_mesh(self, mesh_id, transform_mat):
 
             if attributes.NORMAL is not None and self.apply_normals:
                 normals = self.get_acc_data(attributes.NORMAL)
+                normals = transform.apply_transfomation(normals, transform_mat)
```
Comment (skoudoro) [inline fury/gltf.py:174]: typo to change everywhere: `apply_transformation`

File: fury/gltf.py [hunk 2]
```diff
@@ -77,6 +77,8 @@ def actors(self):
             if self.materials[i] is not None:
                 base_col_tex = self.materials[i]['baseColorTexture']
                 actor.SetTexture(base_col_tex)
+                base_color = self.materials[i]['baseColor']
+                actor.GetProperty().SetColor(tuple(base_color[:3]))
```
Comment (skoudoro) [inline fury/gltf.py:81]: do we have a function in `fury.utils` for doing that?

**Review by skoudoro — COMMENTED** [review]
Hi @xtanion,

See below for some comments

File: fury/gltf.py (hunk 2 above)
Comment (xtanion) [reply fury/gltf.py:81]: No, we have a similar function `set_polydata_colors`

**Comment by skoudoro** [comment]
merging, Thank @xtanion

**Pull request merged**